power_grid_model_c/power_grid_model/include/power_grid_model/component/transformer.hpp

```diff
@@ -258,6 +258,21 @@ class Transformer : public Branch {
             param0.ytt() = y0_series;
         }
 
+        auto const low_admittance = 0.5 * 1e-12 * sn_ / base_power_3p / uk_;
```

Comment:
Should we call this admittance of susceptance? Since we're only adding it to the imaginary part of `Y = G + jB`